imagetest/test_suites/hotattach/hot_attach_test.go

```diff
@@ -215,28 +215,23 @@ func TestFileHotAttach(t *testing.T) {
 			t.Fatalf("unmount failed on linux: %v", err)
 		}
 	}
-	instName, err := exec.Command("hostname").Output()
+	instName, instanceZone, projectNumber, err := utils.GetInstanceZoneProject()
 	if err != nil {
-		t.Fatalf("failed to get hostname: %v %v", instName, err)
+		t.Fatalf("failed to get instance, zone, or project details: error %v", err)
 	}
 
-	projectNumber, instanceZone, err := getProjectAndZone()
```

Comment:
I don't think there's any remaining callers of this function after deleting this, just delete the function too